File: service/src/main/java/ru/diasoft/micro/dofacade/mock/CreateLoanRes.java
package ru.diasoft.micro.dofacade.mock;

import com.fasterxml.jackson.annotation.JsonProperty;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class CreateLoanRes extends InputParams {

    @JsonProperty("LEAD_ID")
    Integer documentID = 234234;

    @JsonProperty("COMMENT")
    String comment = "555\nCOMMENT";

}

File: src/main/java/ru/diasoft/micro/dofacade/json/loan/CreateLoanRes.kt
package ru.diasoft.remote.json.loan

import com.squareup.moshi.Json
import com.squareup.moshi.JsonClass
import ru.diasoft.domain.model.product.ProcessedApplication
import ru.diasoft.micro.dofacade.json.InputParams

@JsonClass(generateAdapter = true)
data class CreateLoanRes(

    @Json(name = "LEAD_ID")
    val documentID: Int,

    @Json(name = "COMMENT")
    val comment: String
) : InputParams()

fun CreateLoanRes.toLoanApplication() = ProcessedApplication(
    //Сумму нам присылают с сервака в том же виде, в котором мы отправляем
    amount = comment.substring(0, comment.indexOf("\n")),
    applicationId = documentID
)